lib/tooltip.js

```diff
@@ -152,7 +152,7 @@ class Tooltip {
     const currentStep = this.currentStepNum();
     const totalSteps = this.tutorial.steps.length;
     this.cta = this.config.cta || (currentStep != totalSteps ? 'Next' : 'Done');
-    this.subtext = this.config.subtext ||
+    this.subtext = typeof this.config.subtext === 'function' ? this.config.subtext :
```

Comment:
I think I'll refactor how we handle config defaults for all the chariot classes by making use of `Object.assign`.  Inspiration: https://github.com/ryanmcdermott/clean-code-javascript